lib/Config.js

```diff
@@ -191,9 +191,39 @@ function hdClientLocationConstraintAssert(configHd) {
     return hdclientFields;
 }
 
+function dmfLocationConstraintAssert(location, locationObj) {
```

Comment:
I don't think it's cloudserver's job to validate (and this be aware of the internals of) dmf locations, as it's never going to use them directly.